peps/pep-8106.rst

```diff
@@ -0,0 +1,91 @@
+PEP: 8106
+Title: Enabling certificate verification by default for stdlib mail modules
+Version: $Revision$
+Last-Modified: $Date$
+Author: Martin Schobert <martin@pentagrid.ch>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 17-Nov-2023
+Sponsor: @vstinner 
```

Comment:
@vstinner could you confirm that you're sponsoring this PEP?

Also, the syntax is wrong, should be something like:
```suggestion
Sponsor: Victor Stinner <vstinner@python.org>
```

Reply:
Fixed